docker-app/qfieldcloud/core/models.py

```diff
@@ -810,6 +810,11 @@ def save(self, *args, **kwargs):
     def teamname(self):
         return self.username.replace(f"@{self.team_organization.username}/", "")
 
+    @staticmethod
+    def format_team_name(organization_name, team_name):
+        if not team_name:
+            raise ValueError("Team name is required.")
+        return f'@{organization_name}/{team_name}'
```

Comment:
```suggestion
            raise ValueError("Team name is required.")

        return f'@{organization_name}/{team_name}'
```

Make sure that before/after `if` blocks and `raise` and `return` keywords there is an empty line.

Reply:
Solved